yt_dlp/extractor/line.py

```diff
@@ -12,7 +12,7 @@
 
 
 class LineTVIE(InfoExtractor):
-    _VALID_URL = r'https?://tv\.line\.me/v/(?P<id>\d+)_[^/]+-(?P<segment>ep\d+-\d+)'
+    _VALID_URL = r'https?://tv\.line\.me/(?:v|embed)/(?P<id>\d+)(?:_[^/]+-(?P<segment>ep\d+(?:-\d+|))|)'
```

Comment:
```suggestion
    _VALID_URL = r'https?://tv\.line\.me/(?:v|embed)/(?P<id>\d+)(?:_[^/]+-(?P<segment>ep\d+(?:-\d+)?))?'
```
Should work the in the same way

Reply:
Works for my list of urls. Is the last ? needed?

Reply:
Actually you are right that last ? is needed